java/create-and-save-map/src/main/java/com/esri/arcgisruntime/sample/createandsavemap/MainActivity.java

```diff
@@ -206,6 +212,7 @@ public void onDrawerOpened(View drawerView) {
    * Inflates the save map dialog allowing the user to enter title, tags and description for their map.
```

Comment:
the method description needs updated here now that the list of folders has been added